README.md

```diff
@@ -7,7 +7,8 @@ http://rawgit.com/nokia/CloudFlow/master/docs/index.html
 ## Features
 * Visualize the flow of workflow executions
 * Identify the execution path of a single task in huge workflows
-* Easily distinguish between simple task (an action) and a sub workflow execution
+* Easily distinguish between simple task (an action) and a sub workflow
+  execution
```

Comment:
Just my OpenStack habit to limit lines by 80 chars. And it seems a little more readable when working with it in the editor in raw format.